extensions/amp-list/0.1/test/integration/test-amp-list.js

```diff
@@ -17,54 +17,58 @@
 import {isExperimentOn} from '../../../../../src/experiments';
 import {poll} from '../../../../../testing/iframe';
 
-const body =
-    '<amp-list id="list" width=300 height=100 ' +
-        'src="http://localhost:9876/list/fruit-data/get?cors=0">' +
-      '<template type="amp-mustache">' +
-        '{{name}} : {{quantity}} @ ${{unitPrice}}' +
-      '</template>' +
-    '</amp-list>';
-
+const TIMEOUT = 12500;
```

Comment:
This appears to be insufficient on Chrome 71. Could it be that something else is going wrong?

Reply:
This was actually a bug. 🙈